Code samples for API usage/sampleCode.py

```diff
@@ -0,0 +1,164 @@
+from dataclasses import dataclass
+import requests
+from time import sleep
+import json
+
+
+# Create a class with attributes that relate to VideoIndexer credentials
+@dataclass
+class VideoIndexer:
+    subscription_key: str = "SUBSCRIPTION KEY GOES HERE"
+    account_id: str = "ACCOUNT ID GOES HERE"
+    location: str = "TRIAL"  # change this if you have a paid subscription tied to a specific location
+
+    @classmethod
+    def get_access_token(cls):
+        """
+        Get an access token from the Video Indexer API. These expire every hour and are required in order to use the
+        service.
+        :return access_token: string.
+        """
+
+        url = "https://api.videoindexer.ai/Auth/{}/Accounts/{}/AccessToken?allowEdit=true".format(
+            cls.location, cls.account_id
+        )
+        headers = {
+            "Ocp-Apim-Subscription-Key": cls.subscription_key,
+        }
+        response = requests.get(url, headers=headers)
+        if response.status_code == 200:
+            access_token = response.json()
+            return access_token
+        else:
+            print("[*] Error when calling video indexer API.")
+            print("[*] Response : {} {}".format(response.status_code, response.reason))
+
+    @classmethod
+    def send_to_video_indexer(cls, video_url, video_id, access_token):
+        """
+        Send a video to be analysed by video indexer.
+        :param video_id: string, identifier for the video..
+        :param video_url: string, public url for the video.
+        :param access_token: string, required to use the API.
+        :return video_indexer_id: string, used to access video details once indexing complete.
+        """
+
+        # Set request headers and url
+        headers = {
+            "Content-Type": "multipart/form-data",
+            "Ocp-Apim-Subscription-Key": cls.subscription_key,
```

Comment:
You don't need to send the subscription key. You are authenticated using the access token